test/script/use-literary-style.js

```diff
@@ -1,11 +1,21 @@
 var expectedResult = 22
 var exec = require('child_process').exec;
-exec('redpen -c test/redpen-conf.xml test/doc/use-literary-style.md -r json', function(err, stdout, stderr){
-  var result = JSON.parse(stdout)
-  if (result[0].errors.length  != expectedResult){
-    console.log(result[0].errors.length + " error was occured. It is unexpected.")
-    process.exit(1);
-  } else {
-    console.log(result[0].errors.length + " error was occured. This is expected.")
-  }
+var fs = require('fs');
+
+var r = fs.createReadStream('use-literary-style.js'),
+    w = fs.createWriteStream('test/use-literary-style.js');
+
+r.pipe(w);
+w.on('close', function(){
+  exec('redpen -c test/redpen-conf.xml test/doc/use-literary-style.md -r json', function(err, stdout, stderr){
+    fs.unlink('test/use-literary-style.js',function(){
+      var result = JSON.parse(stdout)
+      if (result[0].errors.length  != expectedResult){
+        console.log(result[0].errors.length + " error was occured. It is unexpected.")
+        process.exit(1);
+      } else {
+        console.log(result[0].errors.length + " error was occured. This is expected.")
```

Comment:
__[eslint]__
* Error - Unexpected console statement. (no-console)
* Error - 'console' is not defined. (no-undef)